backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/filters/postalcode/PostalCodeProcessor.java

```diff
@@ -0,0 +1,40 @@
+package com.bakdata.conquery.models.datasets.concepts.filters.postalcode;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.univocity.parsers.common.DataProcessingException;
+import com.univocity.parsers.common.ParsingContext;
+import com.univocity.parsers.common.processor.AbstractRowProcessor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+public class PostalCodeProcessor extends AbstractRowProcessor {
+	final private List<Double> data = new ArrayList<>();
+	private int recordsNumber = 0;
+	final private List<String[]> notCorrectRows = new ArrayList<>();
+
+
+	@Override
+	public void processStarted(ParsingContext context) {
+		super.processStarted(context);
+		context.skipLines(1); //skip headers
+	}
```

Comment:
du kannst dir hier den header rausholen und in `rowProcessed` benutzen um nur die relevanten Spalten zu zählen.

Reply:
naja ich weiß eigentlich schon die Reihenfolge der Headers und die csv Datei hat ausschließlich relevante Spalten

Reply:
Es ist keine gute Idee sich auf die Reihenfolge der Spalten in einer CSV zu fixieren. Wir hatten damit schon häufiger Probleme. Die Namen der CSV sollten dafür stabil bleiben. Es ist ja gut möglich, dass sich der weg wie die CSV entsteht ändern wird, aber der Inhalt nicht.